CONTRIBUTING/first-time-contributions.md

```diff
@@ -0,0 +1,83 @@
+# First time contributors
+
+We happily welcome our new contributors to this
+community. If you are contributing to CNCF
+for the first time it is okay if you
+to feel overwhelmed. We as a community
+are always there to help you with any problems
+you are facing. Open-source is about
+collaboration and we are always there
+for supporting each other.
+
+## Getting involved and contributing
+
+As a beginner or first timer, you might find
+difficulties in understanding where to start
+from. Don't worry! We got you.
+
+In the interest of getting more new people
+involved we have issues marked as
+[good-first-issues]. These are issues that have smaller
+scope, but are a great way to start with and
+getting acquainted with the project.
+
+The good-first-issues should also provide
+you details on how to get things resolved or
+how to proceed. If you find it is missing or
+incomplete please tag the person who created
+the issue and let them know.
+
+## Before your first PR
+
+Before you make you first PR, we would like
+you to go through the below attached resources
+for your understanding:
+
+- [How to submit contributions]
+- [Collaborating with pull requests]
+
+## Other ways of communication
+
+You can have additional questions or
+doubt about a certain issue, and we
+believe no doubt should be overlooked.
+You can reach us for help with any doubt
+you have, and we will be happy to have a
+one-on-one discussion.
+
+You can reach us via our [Mailing List].
+
+Join our slack [#tag-security].
+
+### You can reach out to our members
+<!-- cSpell:disable -->
+
+#### STAG CHAIRS
+
+- Emily Fox ([@TheFoxAtWork](https://github.com/TheFoxAtWork))
+- Brandon Lum ([@lumjjb](https://github.com/lumjjb))
+- Aradhana Chetal ([@achetal01](https://github.com/achetal01))
+
+#### Tech Leads
+
+- Justin Cappos ([@JustinCappos](https://github.com/JustinCappos))
+- Ash Narkar ([@ashutosh-narkar](https://github.com/ashutosh-narkar))
+- Andres Vega ([@anvega](https://github.com/anvega))
+
+#### STAG Chair Emeriti
+
+- Dan Shaw ([@dshaw](https://github.com/dshaw))
+- Sarah Allen ([@ultrasaurus](https://github.com/ultrasaurus))
+- Jeyappragash JJ ([@pragashj](https://github.com/pragashj))
+
+## After succesful PR merge
+
+Once you have successfully get your
+first PR merged, you can add your name
+to our Members section in [README.md](README.md).
+
+[good-first-issues]: https://github.com/cncf/tag-security/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22
+[#tag-security]: https://cloud-native.slack.com/archives/CDJ7MLT8S
+[Mailing List]: cncf-tag-security-leads@lists.cncf.io
+[How to submit contributions]: https://opensource.guide/how-to-contribute/#how-to-submit-a-contribution
+[Collaborating with pull requests]: https://docs.github.com/en/github/collaborating-with-pull-requests
```

Comment:
```suggestion
```

Placing the links inline is cleaner.

Reply:
I think there's a personal preference with this... the other side of the argument is that it makes the text more readable especially with long links. Easier `.md` reading formatting for wrapped lines.